src/main/java/com/minecolonies/coremod/entity/ai/citizen/CitizenAI.java

```diff
@@ -175,9 +176,20 @@ private IState calculateNextState()
         }
         else
         {
-            if (citizen.getCitizenSleepHandler().isAsleep() && !citizen.getCitizenDiseaseHandler().isSick())
+            if (citizen.getCitizenSleepHandler().isAsleep())
             {
-                citizen.getCitizenSleepHandler().onWakeUp();
+                if (citizen.getCitizenDiseaseHandler().isSick())
```

Comment:
odd change, before we only did wakeup if not sick, now we do if sick?

Also whats with the null and distance checks before calling it, why do they apply to sick only?